.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.4.0
+  hooks:
+  - id: trailing-whitespace
+    exclude: testing/baselines
+  - id: end-of-file-fixer
+    exclude: testing/baselines
+  - id: check-yaml
+  - id: check-added-large-files
+- repo: https://github.com/psf/black
+  rev: 23.1a1
+  hooks:
+  - id: black
```

Comment:
The black docs say that it is recommended to set a language version here: https://black.readthedocs.io/en/stable/integrations/source_version_control.html  ... it's not clear to me how that matters since Black shouldn't be that selective, and it introduces one more spot where the language version is pinned down. Do you happen to know if it matters?

Reply:
I had to look up the reasoning for that, https://github.com/psf/black/pull/2430. In a quick test this seems to lead to `black` always requiring that the an installation for the given version is available on the host running `pre-commit`. I don't think we want to require that users install python3.7 out of band to be able to use the hooks.

Reply:
Ah, definitely — thanks. Case closed.